gateway-api-spec.yaml

```diff
@@ -1486,6 +1520,7 @@ components:
         message:
           description: The message bytes, hex encoded.
           type: string
+          nullable: true
```

Comment:
Just to check - is this required to pass the test? Does the Gateway API return a null message still, even after my changes?

Reply:
Its actually not required anymore, so I'll remove it